railties/test/generators/app_generator_test.rb

```diff
@@ -776,6 +776,17 @@ def test_quiet_option
     assert_empty output
   end
 
+  def test_quiet_option_quiets_rails_commands
+    generator = generator_class.new(
+      [File.join(destination_root, "myapp")], ["--quiet"], destination_root: destination_root
+    )
+
+    generator.stub :bundle_install?, -> { caller.grep(/run_hotwire/).any? } do
```

Comment:
Is `run_hotwire` a hard dependency here or is there another native command we can look for?
My concern is just that the name may change or be removed and this test will fail, which is unrelated.

Reply:
The only 3 methods that run rails commands by default if `bundle_install?` is true are: `run_hotwire`, `run_solid`, and `run_javascript` (the first 2 being faster). 

I can only choose among them, they don't have a common caller besides being all called by Thor.

I just commit `{ caller.grep(/run_(hotwire|solid)/).any? }` to lower the future chances of failure, but  they are still non zero chances.